datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -839,6 +893,7 @@ mod tests {
         projection: Option<Vec<usize>>,
         schema: Option<SchemaRef>,
         predicate: Option<Expr>,
+        pushdown_predicate: bool,
```

Comment:
Maybe in a follow on PR we can adjust the tests to do the same scan both with and without pushdown